sec-scanners-config.yaml

```diff
@@ -1,6 +1,6 @@
 module-name: istio
 protecode:
-  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:v20231026-42b50b89
+  - europe-docker.pkg.dev/kyma-project/prod/istio-manager:v20231103-4abe83cb
   - europe-docker.pkg.dev/kyma-project/prod/external/istio/install-cni:1.19.0-distroless
```

Comment:
This seems to be outdated, can we also update this to 1.19.3 as part of this PR?